package-lock.json

```diff
@@ -0,0 +1,7012 @@
+{
```

Comment:
It seems that you have used `npm` instead of [https://yarnpkg.com](yarn). Could I get you to reverse that?

Reply:
Sure, will make the suggested changes, and do a bit more testing on the UI as well as only been testing via tests from a calling service.